src/components/card/Card.stories.tsx

```diff
@@ -8,6 +8,21 @@ import { CardBody } from './CardBody/CardBody'
+
+const triangleImage =
```

Comment:
What are your thoughts on moving these to a separate file for storybook constants?

Reply:
I looked into that and wondered what the right approach is. We don't want these included in our final build and I think that would mean they should be excluded in `tsconfig.json`. Should we add `**/*.stories.ts`? Or put this in another `stores.tsx` file that doesn't have any components? Other ideas?

Reply:
Good points. Would it be worth having multiple tsconfig files like [this](https://stackoverflow.com/questions/58461649/how-to-exclude-specific-files-in-typescript-only-for-the-build)?

Our current tsconfig exclusions actually leave us with false positives for import paths for example:
![image](https://user-images.githubusercontent.com/15805554/119888942-f7e94880-bf03-11eb-932c-14524f0156d6.png).

Moving all of those into a `tsconfig.publish.json` or something would resolve that small annoyance as well.

It starts to get funky trying to specify that specific tsconfig file for certain scripts, and that's about where my knowledge drops off.

> should we add `**/*.stories.ts`?

This is the easiest solution, and just requires some diligence with naming convention, but I like it.

Reply:
Hmmm you know what I didn't notice these are at least all on one line within the storybook file. I just saw the wrapping in the github UI. I'm good with it as is for getting this out the door. Its always something we can improve later.

Reply:
Thanks for looking at this. I rebased it on my machine instead of doing that through the GitHub UI and now I need approval again. 😅